src/lib/utils/index.spec.ts

```
@@ -17,7 +14,122 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { resolveCdnUrl, resolveHost, checkOptions, removeEmpty, throttle, range } from './index';
+import * as t from 'tcomb-validation';
 
-test.skip('skip', () => {
-  console.log('tests');
+describe('utils:index', () => {
+  describe('resolveCdnUrl', () => {
+    const session = {
+      apikey: 'DevZpGbhtSzZL5bOtYw7Lz',
+      cname: 'fs.enplug.com',
+      urls: {
+        cdnUrl: 'https://cdn.fs.enplug.com',
+        cloudApiUrl: 'https://cloud.fs.enplug.com',
+        fileApiUrl: 'https://www.fs.enplug.com/api/file',
+        pickerUrl: 'https://static.fs.enplug.com/picker/1.4.4/picker.js',
+        uploadApiUrl: 'https://upload.fs.enplug.com',
+      },
+    };
+    it('should properly resolve cdn url with hashed handle', () => {
+      const handle = '5aYkEQJSQCmYShsoCnZN';
+      const result = resolveCdnUrl(session, handle);
+      expect(result).toEqual('https://cdn.fs.enplug.com');
+    });
+    it('should properly resolve cdn url with src handle', () => {
+      const handle = 'src://test123/flug_8-trans_atlantik-300dpi.jpg';
+      const result = resolveCdnUrl(session, handle);
+      expect(result).toEqual('https://cdn.fs.enplug.com/DevZpGbhtSzZL5bOtYw7Lz');
+    });
+    it('should properly resolve cdn url with hashed handle', () => {
+      const handle = 'https://static1.squarespace.com/static/544eb3cce4b0ef091773611f/t/59ba7ce1bd10f00dcdc80a5f/1505394087367/DSC_0527.jpg';
+      const result = resolveCdnUrl(session, handle);
+      expect(result).toEqual('https://cdn.fs.enplug.com/DevZpGbhtSzZL5bOtYw7Lz');
+    });
+    it('shoudl throw error when using src handle or url without apikey', () => {
```

'should throw an error ..'

---

Fixed